src/components/Tipseen/Tipseen.tsx

```diff
@@ -74,9 +74,9 @@ const Tipseen: VibeComponent<TipseenProps> & {
       hideDelay = 0,
       showDelay = 0,
       title,
-      hideCloseButton = false,
+      hideCloseButton,
```

Comment:
If there is a default value to hideCloseButton we will never check the value of isCloseButtonHidden

Reply:
@hadasfa lets add a test please